### PR TITLE
metrics: fix coprocessor cpu alert

### LINKED ISSUE #13918: coprocessor cpu alert is wrong
### What version of TiKV are you using?

N/A

### What operating system and CPU are you using?

N/A

### Steps to reproduce

No need.

### What did you expect?

The alert name matches the description.

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### What problem does this PR solve?

Issue Number: Close #13918

Name is wrong, and expression isn't accurate when coprocessor threads are configured differently.

### What is changed and how it works?

What's Changed:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests
- No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

File: metrics/alertmanager/tikv.rules.yml [hunk 1]
```diff
@@ -277,17 +277,17 @@ groups:
       value: '{{ $value }}'
       summary: TiKV pending {{ $labels.type }} request is high
 
-  - alert: TiKV_batch_request_snapshot_nums
-    expr: sum(rate(tikv_thread_cpu_seconds_total{name=~"cop_.*"}[1m])) by (instance) / ( count(tikv_thread_cpu_seconds_total{name=~"cop_.*"}) *  0.9 ) / count(count(tikv_thread_cpu_seconds_total) by (instance)) > 0
+  - alert: TiKV_coprocessor_cpu_util
+    expr: sum(rate(tikv_thread_cpu_seconds_total{name=~"cop_.*"}[1m])) by (instance) / (count(tikv_thread_cpu_seconds_total{name=~"cop_."}) by (instance) * 0.9) >= 1
```
Comment (AmoebaProtozoa) [inline metrics/alertmanager/tikv.rules.yml:281]: why is name regex `cop_. ` instead of `cop_.*`?

Comment (tabokie) [reply metrics/alertmanager/tikv.rules.yml:281]: Good catch

**Comment by zjj1002** [comment]
middle_img_v2_2cce6078-785c-45f3-95fd-a980cb3246ag
When we execute such a statement, no data can be found in the monitoring
middle_img_v2_8d142ee0-1308-4cb3-b805-c1936868191g
so, is {name=~"cop_.*"} a correct expression？

**Comment by tabokie** [comment]
@zjj1002 If `readpool.coprocessor.use-unified-pool = true`, coprocessor requests will enter unified read pool instead.

**Review by v01dstar — APPROVED** [review]

**Review by BornChanger — APPROVED** [review]

**Review by youjiali1995 — APPROVED** [review]

**Pull request merged**